docs/sources/static/configuration/integrations/mongodb_exporter-config.md

```diff
@@ -79,6 +79,34 @@ Besides that, there's not much to configure. Please refer to the full reference
   # MongoDB node connection URL, which must be in the [`Standard Connection String Format`](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-standard-connection-string-format)
   [mongodb_uri: <string>]
 
+  # By default all the below stats will be
```

Comment:
This explanation should be also added to flow component doc. But if we change the default behavior, we can remove them altogether.

Reply:
Hi @marctc, there is no change in default behavior. I have just added more configurable parameter.

Reply:
Yes i have already added this to [docs/sources/flow/reference/components/prometheus.exporter.mongodb.md](https://github.com/grafana/agent/pull/6021/files#diff-57174b3cfd9003a73fbd9354096a52c6890fa311a439e0082f50cba8129e281c)

Reply:
By default behavior I meant my comment in https://github.com/grafana/agent/pull/6021/files#r1437078040. The component docs specifies that those flags are disabled by default but then the code does the opposite (as it reflected here).